opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -217,6 +218,14 @@ message DoubleHistogram {
   AggregationTemporality aggregation_temporality = 2;
 }
 
+// DoubleSummary metric data are used to convey quantile summaries,
+// a Prometheus and OpenMetrics data type. These data points cannot
+// always be merged in a meaningful way. While they can be useful in some
+// applications, histogram data points are recommended for new applications.
+message DoubleSummary {
```

Comment:
Prometheus doesn't have support for int summaries but I'm seeing OTLP represents int histograms. Are we planning to do the same here? If yes, it would be good to add a TODO at least.

Reply:
We are not planning to support the int version since this is mostly for backwards compatibility with Prometheus

Reply:
If they add that we will add it